pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -468,6 +467,7 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 
 		s.initCACertsWatcher()
 	}
+	caOpts.OnRootCertUpdate = s.updatePluggedinRootCertAndGenKeyCert
```

Comment:
Can we capture a full rotation and update of the DNS certificates in an integration test?

Reply:
Not seems very viable, because it can take a very long time.

Reply:
Can we modify the root cert validity period?

Reply:
minutes? Seems we cannot even accept minutes, because the security test has already exceed 40minutes IIRC